README.md

```diff
@@ -1,67 +1,38 @@
-# Teleport Plugins and Example Applications
+# Teleport Access Request plugins and example applications
```

Comment:
I would keep the plugins term here.
We have 3 types of plugins and access request is just one of them (the others are event-handler and terraform provider)